infrastructure/tooling/src/changelog/index.js

```diff
@@ -194,7 +194,7 @@ class ChangeLog {
         const snippets = categorizedSnippets[audience]
           .map(({ level, reference, body }) => (
             '▶ ' + levelLabels[level] +
-            (reference ? reference : '') + '\n' +
+            (reference ? ' ' + reference : '') + '\n' +
```

Comment:
I think leading space was ok? to pad string to the '>' above?

Reply:
This adds the space after the level in the case where there is a reference to an issue/bug

Reply:
You can see it prints `levelLabels[level]` first. This extra space is for after that.